mypy/types.py

```diff
@@ -1488,3 +1490,49 @@ def true_or_false(t: Type) -> Type:
     new_t.can_be_true = type(new_t).can_be_true
     new_t.can_be_false = type(new_t).can_be_false
     return new_t
+
+
+def replace_none(n: Node) -> Node:
+    """
+    Helper for parse and fastparse.
+    Replace all Nones on the right of protected assignment with Any-like.
```

Comment:
I don't see the term "protected assignment" in PEP 484.  Could you define it here?